python/skupper_router/management/skrouter.json

```diff
@@ -1282,6 +1304,29 @@
                     "type": "integer",
                     "graph": true,
                     "description": "The number of bytes sent from clients to servers on all connections created by this connector."
+                },
+                "encryptedBytesIn": {
+                    "type": "integer",
+                    "graph": true,
+                    "description": "The number of encrypted bytes sent from servers to clients on all connections created by this connector. This will be zero if this tcpConnector does not have an associated sslProfile."
+                },
+                "encryptedBytesOut": {
+                    "type": "integer",
+                    "graph": true,
+                    "description": "The number of encrypted bytes sent from clients to servers on all connections created by this connector. This will be zero if this tcpConnector does not have an associated sslProfile."
+                },
+                "sslProfile": {
+                    "type": "string",
+                    "required": false,
+                    "description": "Name of the sslProfile.",
+                    "create": true
+                },
+                "verifyHostname": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "yes: Ensures that when initiating a connection (as a client) the host name in the URL to which this connector connects to matches the host name in the digital certificate that the peer sends back as part of the TLS connection; no: Does not perform host name verification",
+                    "deprecationName": "verifyHostName",
```

Comment:
remove: this is a new attribute that never existed before so there was no name to deprecate, right?

Reply:
Removed the deprecationName attribute